src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/WindowsForms/WindowsFormsEditorProvider.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.ProjectSystem.Properties;
+using System;
+using System.ComponentModel.Composition;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.WindowsForms
+{
+    /// <summary>
+    ///     A project-specific editor provider that is responsible for handling two things;
+    ///     
+    ///     1) Add the Windows Forms designer the list of editor factories for a "designable" source file, and 
+    ///        determines whether it opens by default.
+    ///     
+    ///     2) Persists whether the designer opens by default when the user uses Open With -> Set As Default.
+    /// </summary>
+    [Export(typeof(IProjectSpecificEditorProvider))]
+    [AppliesTo(ProjectCapability.DotNet)]
```

Comment:
Unsure yet what capability to tie this lightup to.

Reply:
`DotNet` in not in this list - https://github.com/microsoft/VSProjectSystem/blob/master/doc/overview/project_capabilities.md, what projects have it, both core and classic?

Reply:
Classic does not have .NET, but it doesn't matter this code never runs in the case of a legacy project system as it is CPS specific.

Reply:
By default .NET classic projects use the legacy project system, however we can create a CPS-based project  manually and it works fine. So this will disable double-click in these projects and we say that this is OK because they are not supported officially and we'll face it later if needed?